docs/pages/admin-guides/access-controls/access-requests/resource-requests.mdx

```diff
@@ -291,387 +287,15 @@ Getting updated certificates...
-
-#### `node`
-
-You can restrict access to searching `node` resources by assigning values to the
-`node_labels` field in the `spec.allow` or `spec.deny` fields. The following
-role allows access to SSH Service instances with the `env:staging` label.
-
-```yaml
-kind: role
-version: v5
-metadata:
-  name: staging-access
-spec:
-  allow:
-    node_labels:
-      env: staging
-    logins:
-      - "{{internal.logins}}"
-  options:
-    # Only allows the requester to use this role for 1 hour from time of request.
-    max_session_ttl: 1h
-```
-
-#### `kube_cluster`
-
-You can restrict access to searching `kube_cluster` resources by assigning
-values to the `kubernetes_labels` field in the `spec.allow` or `spec.deny`
-fields.
-
-The following role allows access to Kubernetes clusters with the `env:staging`
-label:
-
-```yaml
-kind: role
-metadata:
-  name: kube-access
-version: v7
-spec:
-  allow:
-    kubernetes_labels:
-      'env': 'staging'
-    kubernetes_resources:
-      - kind: '*'
-        namespace: '*'
-        name: '*'
-  deny: {}
-```
-
-#### Kubernetes resources
-
-You can restrict access to Kubernetes resources by assigning values to the
-`kubernetes_resources` field in the `spec.allow` or `spec.deny` fields.
-
-The following role allows access to Kubernetes pods with the name `nginx` in any
-namespace, and all pods in the `dev` namespace:
-
-```yaml
-kind: role
-metadata:
-  name: kube-access
-version: v7
-spec:
-  allow:
-    kubernetes_labels:
-      '*':'*'
-    kubernetes_resources:
-      - kind: pod
-        namespace: "*"
-        name: "nginx*"
-      - kind: pod
-        namespace: "dev"
-        name: "*"
-    kubernetes_groups:
-      - viewers
-  deny: {}
-```
-
-##### Preventing unintended access to Kubernetes resources
-
-If you are setting up a Teleport role to enable just-in-time access to a
-specific Kubernetes resources, you should set the role's `kubernetes_groups` and
-`kubernetes_users` to a role that has no access to Kubernetes resource beside
-the Kubernetes resources that Teleport is able to restrict access for.
-
-This is because, if a user requests access to a Kubernetes pod, and the request
-is approved, the Teleport Kubernetes Service will use the `kubernetes_groups`
-and `kubernetes_users` fields in the role to add impersonation headers to the user's
-requests to a Kubernetes API server. Under these conditions, Teleport will be able
-to restrict access to all Kubernetes resources kinds mentioned above except for
-the desired `pod`. Teleport is also able to restrict access to namespaced-scoped
-custom resources but not cluster-scoped custom resources - CRDs resources that
-are cluster-scoped will be accessible to the user if the principals in the
-`kubernetes_users` and `kubernetes_groups` fields have access to them.
-
-Requesting access to a Kubernetes Namespace allows you to access all resources
-in that namespace but you won't be able to access any other supported resources
-in the cluster.
```

Comment:
This section was removed rather than moved but it seems important, no? I can see how the `node` and `kube_cluster` sections were unnecessary, but is the information contained within this section expressed anywhere else in the docs?

Reply:
Addressed in e00f238043fe14e47775ecadb07909e1ab9c7cbf